### PR TITLE
Renaming Classes, Changing MasterID and MasterInfo

### PR DESCRIPTION
**Pull request by eavivi4**
Changing classes that contain Master/SlavePort and changing MasterID and MasterInfo into UniqueID and RequestorInfo respectively.

### CODE REVIEW

**Comment by powerjg** [comment]
@shparekh00 and @LakinS I've tagged you as reviewers. Could at least one of you review these changes? Thanks!

File: src/dev/arm/smmu_v3_ports.cc [hunk 1]
```diff
@@ -138,7 +138,7 @@ SMMUControlPort::getAddrRanges() const
 
 SMMUATSMasterPort::SMMUATSMasterPort(const std::string &_name,
```
Comment (shparekh00) [inline src/dev/arm/smmu_v3_ports.cc]: Do we need to change this class too?

File: src/dev/x86/intdev.hh [hunk 2]
```diff
@@ -94,7 +94,7 @@ buildIntPacket(Addr addr, T payload)
 }
 
 template <class Device>
-class IntMasterPort : public QueuedMasterPort
+class IntMasterPort : public QueuedRequestPort
```
Comment (shparekh00) [inline src/dev/x86/intdev.hh]: Should this be IntRequestPort?

File: src/mem/cache/base.cc [hunk 3]
```diff
@@ -66,7 +66,7 @@ using namespace std;
 BaseCache::CacheSlavePort::CacheSlavePort(const std::string &_name,
```
Comment (shparekh00) [inline src/mem/cache/base.cc]: CacheSlavePort -> CacheResponsePort

File: src/mem/cache/base.hh [hunk 4]
```diff
@@ -119,7 +119,7 @@ class BaseCache : public ClockedObject
      * and the sendDeferredPacket of the timing port is modified to
      * consider both the transmit list and the requests from the MSHR.
      */
-    class CacheMasterPort : public QueuedMasterPort
+    class CacheMasterPort : public QueuedRequestPort
```
Comment (shparekh00) [inline src/mem/cache/base.hh]: CacheRequestPort

File: src/mem/coherent_xbar.hh [hunk 5]
```diff
@@ -82,7 +82,7 @@ class CoherentXBar : public BaseXBar
      * be instantiated for each of the master ports connecting to the
      * crossbar.
      */
-    class CoherentXBarSlavePort : public QueuedSlavePort
+    class CoherentXBarSlavePort : public QueuedResponsePort
```
Comment (shparekh00) [inline src/mem/coherent_xbar.hh]: I think CoherentXBarSlavePort needs to be changed

**Pull request closed without merging**